topics/cluster-spec.md

```diff
@@ -513,6 +513,37 @@ set the slots to their normal state again. The same command is usually
 sent to all other nodes to avoid waiting for the natural
 propagation of the new configuration across the cluster.
 
+#### Synchronous replication of `CLUSTER SETSLOT`
+
+Starting from Valkey 8.0, the `CLUSTER SETSLOT` command is synchronously replicated to all healthy replicas
+running Valkey version 8.0+. By default, this synchronous replication must complete within 2 seconds.
+If the replication fails, the primary does not execute the command, and the client receives a
+`NOREPLICAS Not enough good replicas to write` error. Operators can retry the command or customize the
+timeout using the `TIMEOUT` parameter to further increase the reliability of live reconfiguration:
```

Comment:
```suggestion
timeout using the `TIMEOUT` parameter to further increase the reliability of live resharding:
```
Everywhere else in this doc we call it live resharding and not "live reconfiguration". I think we should be consistent.

Reply:
I picked this phrase (live reconfiguration) from the section title https://github.com/valkey-io/valkey-doc/pull/167/files#diff-93c993e7eca8dc9073483f2c2bc33f13fd9ebd7c2a3a3657b364bca84d5f289eL406 :)

This is also pretty much the first time I came across this terminology. Any concern if I update the section title to "live resharding" too for consistency?

Reply:
I guess "reconfiguration" includes adding and removing nodes. Resharding is only about moving slots?

I'm fine with either of the terms. :)

Reply:
Yes, but these concepts are intertwined. We mainly move slots due to node arrivals or departures. :)  While I slightly prefer 'resharding' at the moment, 'reconfiguration' also works for me. I do feel strongly though about consistency.